docs/data/sdk-debuggability.md

```diff
@@ -0,0 +1,36 @@
+
+## Event Dropping
+
+If you are not able to ingest any event, there following might be the possible reasons:
+
+- if you are using the right API key. If you have enabled data residency in the EU, you'll need to retrieve your API key from `https://analytics.eu.amplitude.com/`. This is where your specific API details are located due to data locality regulations.
+
+- If you are using the right instance. Ensure that you're using the correct instance. If multiple versions of SDKs exist, they might cause conflict issues. To avoid this, please provide different instance names for each instance. For the latest SDKs, you might need to create separate variables in order to instantiate them differently.
+
+- If you have a project in Data, please ensure that the event hasn't been blocked.
+
+- Please check if your deviceId or userId are valid. [More details](../../analytics/apis/http-v2-api/#device-ids-and-user-ids-minimum-length).
+
+## Privacy 
+
+If you've already disabled IP, it's still possible to see the IP in your user lookup if you're using the [latest SDK](../sdks/sdk-architecture/). We send the data to the HTTP API (HTTP API V1 for maintenance SDK and HTTP API V2 for the latest SDK). If you disabled the IP address midway, it's possible that the user's previous IP address was saved in our backend. Our backend will retrieve the IP from the database, if there's any. If it's a test user, it's probably fine. It won't affect incoming new users after you disable the IP. If this affects all users, you might need to create a new workspace.
+
+## client_upload_time shows a future time
+
+Check this section if you're seeing `client_upload_time` appearing as a time in the future. The `client_upload_time` is determined by the customer's device. It's possible that it may show a time in the future if the customer's clock is incorrectly set. You can remove the time in the Event payload via the Enrichment Plugin if you are using latest SDK. This will stop using the customer's device clock and instead rely on server_upload_time. However, be aware that this approach has a downside. If events are not uploaded immediately, the recorded event time can differ significantly from the original time the event was fired.
+
+## Device family is not appropriate
+
+We are using a [third party libray](https://github.com/faisalman/ua-parser-js) library to parse the info for all Browser SDKs, except for @amplitude/analytics-browser@^2.0. For other SDKs, we rely on server device mapping. We refer to [this resource](http://storage.googleapis.com/play_public/supported_devices.html), [this resource](https://en.wikipedia.org/wiki/List_of_Android_smartphones) for Android, and [this resource](https://en.wikipedia.org/wiki/Comparison_of_tablet_computers) for iOS. If you find an inappropriate device family that doesn't exist in any of these files, please submit a ticket here. Please note that updates to device mapping on our side will only affect incoming events.
```

Comment:
- this is one of resource for Android, please check [here](https://amplitude.atlassian.net/wiki/spaces/GOV/pages/2087223297/Device+Mapping) for more reference.
- I didn't mention it right now because we don't have a fix for that yet, what should we say related to this issue?  We might also need to remember to update if we have some solutions.

Reply:
It would reduce more ask about it If we could add a note/banner here saying something like "if you notice K device family, it's expected because of Google's UA reduction (link to the article). Amplitude might update it if the reduction affects more fields"
 
@justin-fiedler wdyt?